couchpotato/core/media/movie/providers/info/omdbapi.py

```diff
@@ -88,8 +88,9 @@ def parseMovie(self, movie):
 
             tmp_movie = movie.copy()
             for key in tmp_movie:
-                if tmp_movie.get(key).lower() == 'n/a':
-                    del movie[key]
+                if type(tmp_movie.get(key)) is string:
```

Comment:
This is a bad practice. If anything you'd need to use ```isinstance``` with basestring on python 2 and just string on python 3 but it's likely you'll just want to check if it's true-ish or just catch a KeyError.